cfme/tests/containers/test_containers_properties.py

```diff
@@ -40,7 +41,8 @@ def test_pods_properties_rel(provider, rel):
     the fields in the Properties table
     """
     navigate_to(Pod, 'All')
-    ui_pods = [r.name.text for r in list_tbl_pods.rows()]
+    ui_pods = sample([r.name.text for r in list_tbl_pods.rows()],
```

Comment:
Updating pytest for cm-env1 uncovered a bug in this - was your local testing successful? list_tbl.pods.rows() is a generator - check out web_ui.Table.row_count, which uses Table.rows_as_list() and returns the length.